library/src/helpers/schema.ts

```diff
@@ -200,6 +182,73 @@ export class SchemaHelpers {
     return new SchemaClass(json);
   }
 
+  static getCustomExtensions(value: any) {
```

Comment:
Please provide at least jsdoc for the function 🙂

Reply:
I don't put any JSDocs for these functions and also for another, because we only export AsyncAPI component from this package, so this functionality is private and the moment only used in one place :)

Reply:
Done see it as only for the external user, but also for us maintainers, so we can quickly get an overview over what the functions do.